lib/index.js

```diff
@@ -13,8 +16,13 @@ exports.register = (server, baseOptions, next) => {
 
         return Async.each(list, (file, callback) => {
 
-            const path = process.cwd() + '/' + baseOptions.dir + '/' + file;
-            const routeFile = require(path);
+            // Skip files with unknown filename extension
+            if (!file.match(baseOptions.test || testDefault)) {
```

Comment:
While this is a possible breaking change I think it is a minor one and what you would expect. 👍